### PR TITLE
Update AWS gv config

### PR DESCRIPTION
**Pull request by ykalidin**
Updated AWS gv configuration to use AWS ECS task credentials instead of providing them explicitly.

### CODE REVIEW

**Review by nareshkumarthota — APPROVED** [review]
LGTM

File: cloud/docker/gvproviders/custom/aws/run.sh [hunk 1]
```diff
@@ -5,28 +5,36 @@
...
-  echo "ERROR: Cannot read GVs from AWS Secrets Manager.."
-  echo "ERROR: Specify env variable AWS_DEFAULT_REGION"
-  exit 1
+if [[ ! -z "$AWS_CONTAINER_CREDENTIALS_RELATIVE_URI" ]]; then
+    json=$(curl 169.254.170.2$AWS_CONTAINER_CREDENTIALS_RELATIVE_URI)
+    AWS_ACCESS_KEY_ID=$(echo "$json" | /home/tibco/be/gvproviders/jq -r '.AccessKeyId')
+    #AWS_ROLE_ARN=$(echo "$json" | /home/tibco/be/gvproviders/jq -r '.RoleArn')
+    AWS_SECRET_ACCESS_KEY=$(echo "$json" | /home/tibco/be/gvproviders/jq -r '.SecretAccessKey')
+    AWS_SESSION_TOKEN=$(echo "$json" | /home/tibco/be/gvproviders/jq -r '.Token')
+else
```
Comment (rameshpolishetti) [inline cloud/docker/gvproviders/custom/aws/run.sh:14]: @ykalidin any specific reason for keeping `else` part? 
I feel `if` can be closed here.

Comment (ykalidin) [reply cloud/docker/gvproviders/custom/aws/run.sh]: Updated accordingly

**Review by rameshpolishetti — APPROVED** [review]
Looks good.

**Comment by LakshmiMekala** [comment]
Validated aws gv config in aws ecs -> passing aws_secret_id to env in task and its working fine.
Also validated running docker image locally passing envs as awskey, secret_key and secret manager id, working fine.

**Pull request merged**